.drone.yml

```diff
@@ -6043,7 +6043,94 @@ volumes:
     claim:
       name: drone-s3-debrepo-pvc
 ---
+kind: pipeline
```

Comment:
Isn't .drone.yml auto generated? Where did the changes came from?

Reply:
Not everything is generated by Dronegen. Stuff like `promote-build`

https://github.com/gravitational/teleport/blob/baa2bb2e12b7bec492225aa79784375b1e18bdb4/.drone.yml#L5893

or these:

https://github.com/gravitational/teleport/blob/baa2bb2e12b7bec492225aa79784375b1e18bdb4/dronegen/cron.go#L17-L20
https://github.com/gravitational/teleport/blob/baa2bb2e12b7bec492225aa79784375b1e18bdb4/dronegen/misc.go#L17-L20

are managed manually.

Dronegen does not overwrite the entire `.drone.yml`, it merges what it manages with literal pipelines, as seen here:
https://github.com/gravitational/teleport/blob/baa2bb2e12b7bec492225aa79784375b1e18bdb4/dronegen/main.go#L51-L54

Reply:
Thanks.